Candidate: CVE-2022-42969
PublicDate: 2022-10-16 06:15:00 UTC
References:
 https://github.com/pytest-dev/py/issues/287
 https://github.com/pytest-dev/py/issues/287#issuecomment-1283784766
 https://www.cve.org/CVERecord?id=CVE-2022-42969
Description:
 ** DISPUTED ** The py library through 1.11.0 for Python allows remote
 attackers to conduct a ReDoS (Regular expression Denial of Service) attack
 via a Subversion repository with crafted info data, because the
 InfoSvnCommand argument is mishandled. Note: This has been disputed by
 multiple third parties as not being reproduceable and they argue this is
 not a valid vulnerability.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]
 mitre: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L [5.3 MEDIUM]

Patches_py:
upstream_py: needs-triage
trusty_py: ignored (end of standard support)
xenial_py: ignored (end of standard support)
bionic_py: DNE
focal_py: DNE
esm-infra/focal_py: DNE
jammy_py: DNE
kinetic_py: DNE
lunar_py: DNE
mantic_py: DNE
noble_py: DNE
devel_py: DNE

Patches_python-py:
upstream_python-py: needs-triage
trusty_python-py: ignored (end of standard support)
xenial_python-py: ignored (end of standard support)
esm-apps/xenial_python-py: not-affected (disputed)
bionic_python-py: ignored (end of standard support, was needs-triage)
esm-apps/bionic_python-py: not-affected (disputed)
focal_python-py: not-affected (disputed)
esm-infra/focal_python-py: DNE (focal was not-affected [disputed])
esm-apps/focal_python-py: not-affected (disputed)
jammy_python-py: not-affected (disputed)
esm-apps/jammy_python-py: not-affected (disputed)
kinetic_python-py: ignored (end of life, was needs-triage)
lunar_python-py: ignored (end of life, was needs-triage)
mantic_python-py: ignored (end of life, was needs-triage)
noble_python-py: not-affected (disputed)
esm-apps/noble_python-py: not-affected (disputed)
devel_python-py: not-affected (disputed)
